doc/faq.rst

```diff
@@ -300,6 +300,32 @@ reviewers are busy. We ask for your understanding and request that you
 not close your pull request or discontinue your work solely because of
 this reason.
 
+For tips on how to make your pull request easier to review and more likely to be
+reviewed quickly, see :ref:`improve_issue_pr`.
+
+.. _improve_issue_pr:
+
+How do I improve my issue or pull request?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To help your issue receive attention or your pull request be more likely to
+be reviewed, you can try:
+
+* follow our :ref:`contribution guidelines <contributing>`, in particular
+  :ref:`automated_contributions_policy`, :ref:`filing_bugs`,
+  :ref:`stalled_pull_request` and :ref:`stalled_unclaimed_issues`.
```

Comment:
This is probably not ideal but as mentioned above, our contributing page is quite long at the moment and contains a lot of info that would not be relevant in this case. Thus, I thought it would be nice to point to relevant sections.
It would be nice if there was a single page to point (new) contributors to, guiding them on etiquette and general info on making a good issue/PR, but that would require re-organisation of the contributing page, which would be a big task.